### ISSUE TITLE
Additional Types of Encryption

### ISSUE THREAD

**Issue by DogeDevYT** [hole-project/hole#6]
**Is your feature request related to a problem? Please describe.**
No problem just d1snin is a big fat meanine stupid head and wont' let me contribute because I'll "mess up his code" and 
"ruin hours worth of conventions"

**Describe the solution you'd like**
I want to have more encryption algorithms available for me to use and the ability to choose in a simplified manner. For example if you're initializing the program through command line an additional argument to specify type of encryption.

**Describe alternatives you've considered**
I literally don't care.

**Additional context**
![Anime_Training_Protocol_Compressed](https://user-images.githubusercontent.com/52612977/172618397-5e3c7714-b7f1-4707-9782-76a69f4f85a0.png)
![doge crying](https://user-images.githubusercontent.com/52612977/172618449-df07234a-dcf6-4d30-9f25-591851e946a4.png)
![gigachad in fedora](https://user-images.githubusercontent.com/52612977/172618469-c228b2be-4019-4e49-9d7b-978aceab1668.png)
![no kotlin](https://user-images.githubusercontent.com/52612977/172618493-d142a6a2-16f4-4575-aeef-ba351f15494f.png)
![no room for commies only lasangna](https://user-images.githubusercontent.com/52612977/172618502-cfa8fce8-1921-4994-8c7f-81d8e2462a11.jpg)
![oi josuke i have been rejected by 5 girls this week and am having a mental breakdown](https://user-images.githubusercontent.com/52612977/172618520-98262f90-2c7d-48d3-946c-21311f8caf4a.png)
![sperm whale simp card](https://user-images.githubusercontent.com/52612977/172618564-6b8e7d7a-6859-46d2-9070-32573be05642.png)
![yandere dev coding be like](https://user-images.githubusercontent.com/52612977/172618660-56999700-aaeb-4af5-a68e-ed67d77f5bd7.PNG)

**Comment by DogeDevYT** [comment]
I would also like to use gpg encryption algorithm as I intend to use this to store my top secret workout routine on a remote linux desktop with an obscure version of arch.

**Comment by DogeDevYT** [comment]
damn aight

**Comment by DogeDevYT** [comment]
what about making a new encryption algorithm for security by obscurity. I could try making that

**Comment by DogeDevYT** [comment]
I would call it basedencrypt

**Comment by DogeDevYT** [comment]
incomprehensible, I've been met with a terrible fate.

**Comment by d1snin** [comment]
yeah and ive already finished making a pipebomb with an engraved greeting with your name

**Comment by DogeDevYT** [comment]
I am going to fucking obliterate you painfully and slowly. I will catch you in 4k UHD surround sound 16 Gigs ram, HDR GEFORCE RTX, TI-80 texas insturments, Triple A duracell battery ultrapower100 Cargador Compatible iPhone 1A 5 W 1400 + Cable 100% 1 Metro Blanco Compatible iPhone 5 5 C 5S 6 SE 6S 7 8 X XR XS XS MAX GoPro hero 1 2 terrabyte xbox series x Dell UltraSharp 49 Curved Monitor - U4919DW Sony HDC-3300R 2/3" CCD HD Super Motion Color Camera, 1080p Resolution Toshiba EM131A5C-SS Microwave Oven with Smart Sensor, Easy Clean Interior, ECO Mode and Sound On/Off, 1.2 Cu. ft, Stainless Steel HP LaserJet Pro M404n Monochrome Laser Printer with Built-in Ethernet (W1A52A) GE Voluson E10 Ultrasound Machine LG 23 Cu. Ft. Smart Wi-Fi Enabled InstaView Door-in-Door Counter-Depth Refrigerator with Craft Ice Maker GFW850SPNRS GE 28" Front Load Steam Washer 5.0 Cu. Ft. with SmartDispense, WiFi, OdorBlock and Sanitize and Allergen - Royal Sapphire Kohler K-3589 Cimarron Comfort Height Two-Piece Elongated 1.6 GPF Toilet with AquaPiston Flush Technology., Quick Charge 30W Cargador 3.0 Cargador de Viaje Enchufe Cargador USB Carga Rápida con 3 Puertos carga rápida Adaptador de Corriente para iPhone x 8 7 Xiaomi Pocophone F1 Mix 3 A1 Samsung S10 S9 S8AUKEY Quick Charge 3.0 Cargador de Pared 39W Dual Puerto Cargador Móvil para Samsung Galaxy S8 / S8+/ Note 8, iPhone XS / XS Max / XR, iPad Pro / Air, HTC 10, LG G5 / G6 AUKEY Quick Charge 3.0 Cargador USB 60W 6 Puerto Cargador Móvil para Samsung Galaxy S8 / S8+ / Note 8, LG G5 / G6, Nexus 5X / 6P, HTC 10, iPhone XS / XS Max / XR, iPad Pro/ Air, Moto G4 SAMSUNG 85-inch Class Crystal UHD TU-8000 Series - 64K UHD HDR Smart TV with Alexa Built-in (UN85TU8000FXZA, 2020 Model) GE 38846 Premium Slim LED Light Bar, 18 Inch Under Cabinet Fixture, Plug-In, Convertible to Direct Wire, Linkable 628 Lumens, 3000K Soft Warm White, High/Off/Low, Easy to Install, Easy to Install, 18 Ft Bissell Cleanview Swivel Pet Upright Bagless Vacuum Cleaner Trane. This is not a joke, this is my serious intention. 
![simp card](https://user-images.githubusercontent.com/52612977/172643236-ac5a8788-e127-41a3-ae2b-dc839ffed5b0.jpg)
![what is wrong with you](https://user-images.githubusercontent.com/52612977/172643308-60ee63ea-a579-4824-a0a8-98705b98b7ea.jpg)
![wide ethan](https://user-images.githubusercontent.com/52612977/172643405-06c755c3-1f95-4890-bf02-af60d75aed49.png)
![say gn and go to bed at a reasonable hour](https://user-images.githubusercontent.com/52612977/172643504-ff845159-7594-45f5-9781-b27944b6d1f6.png)
![shitass](https://user-images.githubusercontent.com/52612977/172643507-b2345067-bed3-46c1-a614-d255ebcfd33a.png)
![uncle iroh murphy](https://user-images.githubusercontent.com/52612977/172643509-045257c6-43d5-4bf0-a8aa-aeb3fbaa78c0.png)
![we going international](https://user-images.githubusercontent.com/52612977/172643517-51af5bdc-c9ec-441e-86d7-fdcca7c5956a.png)
![woe](https://user-images.githubusercontent.com/52612977/172643518-41430469-843d-4df7-bbcb-f0599713f7ba.gif)
